hal_st/stm32fxxx/TimerStm.hpp

```diff
@@ -46,16 +46,17 @@ namespace hal
         TimerBaseStm(uint8_t aTimerIndex, Timing timing, const Config& config);
         ~TimerBaseStm();
 
-        void Start();
-        void Stop();
+        virtual void Start();
+        virtual void Stop();
+
+    protected:
+        TIM_HandleTypeDef handle = {};
 
     private:
         void ConfigureTrigger();
 
-    private:
         uint8_t timerIndex;
```

Comment:
I'd like to separate blocks of functions and blocks of variables by an access declaration

```suggestion
    private:
        uint8_t timerIndex;
```